docs/repos-feature-usage.html

```diff
@@ -23,3 +23,14 @@ <h3>Repository Feature Usage</h3>
 		</p>
 	</div>
 </div>
+
+<div class="chart-placeholder">
+	<h3>Most Active Repositories</h3>
+	<table
+		data-url="{{ site.dataURL }}/repository-usage-detailed.tsv"
+		data-config='{"slice": [0, 50]}'
+	></table>
+	<div class="info-box">
+		<p>The top 50 most active repositories, by number of pushes.</p>
```

Comment:
Is the `,` here correct?

Reply:
I’d say it’s a matter of preferences. Strictly, it isn’t necessary, but a comma can be used to separate additional information (in this case, “by number of pushes”) from the rest of the sentence. In this simple case, I’d leave the comma away.